### DISCUSSION TITLE
Use consistent parameter name for file system type (fs_type vs fstype)

### DISCUSSION THREAD

**Discussion by jinm** [saltstack/salt#67521] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
mount uses 'fstype'. blockdev uses 'fs_type'

**Describe the solution you'd like**
use 'fs_type'

**Comment by Akm0d** [comment]
To clarify,  [blockdev](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.blockdev.html) uses `fs_type` and [mount](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.mount.html) uses `fstype` and the request is to change the mount state to also call the kwarg `fs_type`?

**Comment by jinm** [comment]
Yes, thanks.

**Comment by Akm0d** [comment]
This would be a no-brainer to do when the two states were initially being created, but now I'd be worried about it breaking existing states -- for cosmetic reasons.  If there's high demand for the change let's do it, but we'll need to see more support for this feature request :+1:

**Comment by jinm** [comment]
Totally agree with your point.

**Comment by jinm** [comment]
Can we consider making 'fs_type' as a valid input for the mount module in a future release and eventually retire 'fstype' couple releases away?

**Comment by Akm0d** [comment]
Yes, that would be a good way to do it :+1:
